### ISSUE TITLE
Error while generating bindings.

### ISSUE THREAD

**Issue by redsett** [HxGodot/hxgodot#57]
I get this error when attempting to generate bindings.

Windows 10
haxe 4.3.2
hxcpp 4.3.2
compiletime 2.8.0

```
Generating binding classes for Godot Engine v4.2.dev.custom_build (float_64)...
Targeting "D:/Personal/Godot/HaxeTest/bindings"
(unknown) : UNSUPPORTED UNARY OPERATOR FOUND! operator_NOT_null
D:\HaxeToolkit\haxe\std/haxe/macro/Printer.hx:390: characters 19-25 : Uncaught exception Null Access
src/godot/macros/ClassGenMacros.hx:1164: characters 26-58 : Called from here
src/godot/macros/ClassGenMacros.hx:62: characters 9-40 : Called from here
(unknown) : Called from here
```

**Comment by dazKind** [comment]
Hi there,

there seems to be an issue with haxe 4.3.2. For the moment you can use haxe 4.3.1 and it should work.

**Comment by dazKind** [comment]
I found the issue with 4.3.2 and filed it here: https://github.com/HaxeFoundation/haxe/issues/11307

**Comment by dazKind** [comment]
Ok, this will get fixed in 4.3.3. In the meantime I deployed a version fix so things should work fine now. Thanks for reporting this! :)

**Comment by redsett** [comment]
Awesome! Thanks for the quick fix!
